### PR TITLE
Fixed some small errors in CSS for modules and some errors in emulators in general.

### PR DESCRIPTION
**Pull request by favorit5503**
Fixed some errors in emulators for the GM module - With Legion and BFA the GM module wont error out now.
Fixed some errors in emulator itself - Bad expansion naming
Fixed some CSS errors for the modules. - Wrong button sizes, etc.

### CODE REVIEW

File: application/emulators/Trinity_rbac_wod_soap.php [hunk 1]
```diff
@@ -27,8 +27,7 @@ class Trinity_rbac_wod_soap implements Emulator
 	 * Array of expansion ids and their corresponding names
 	 */
 	protected $expansions = array(
-		8 => "Battle for Azeroth",
-		7 => "Legion Allied Races",
+		7 => "Battle for Azeroth",
```
Comment (mdjong1) [inline application/emulators/Trinity_rbac_wod_soap.php:30]: This kind of change doesn't belong in a CSS error fix PR. Make another PR for removing Legion Allied Races.

File: application/emulators/trinity_rbac_bfa_soap.php [hunk 2]
```diff
@@ -50,7 +50,7 @@ class Trinity_rbac_bfa_soap implements Emulator
 		"character_stats" => "character_stats",
 		"guild_member" => "guild_member",
 		"guild" => "guild",
-		"gm_tickets" => "gm_ticket"
+		"gm_tickets" => "gm_bug"
```
Comment (mdjong1) [inline application/emulators/trinity_rbac_bfa_soap.php:53]: Is this related to the CSS fix?

**Comment by favorit5503** [comment]
So you didnt see what i wrote to this PR. I made these 3 changes:
Fixed some CSS errors for the modules.
Fixed some errors in emulators for the GM module.
Fixed some errors in emulator itself.

**Comment by mdjong1** [comment]
I still think CSS changes and removing a whole expansion are different PR's

**Comment by alexandre433** [comment]
I agree, but Legion Allied Races, isn't a expasion, should just be Legion

**Comment by favorit5503** [comment]
> I agree, but Legion Allied Races, isn't a expasion, should just be Legion

Yes and this is why i am fixing this

**Comment by mdjong1** [comment]
> I agree, but Legion Allied Races, isn't a expasion, should just be Legion

Should still be a separate PR regardless :p

But has been noted that it will be separated in the future
image
